src/properties.json

```diff
@@ -1,6 +1,6 @@
 {
     "matlabBatchUrl": "https://ssd.mathworks.com/supportfiles/ci/matlab-batch/v0/install.sh",
     "matlabDepsUrl": "https://ssd.mathworks.com/supportfiles/ci/matlab-deps/v0/install.sh",
-    "matlabLatestReleaseUrl": "https://ssd.mathworks.com/supportfiles/ci/matlab-release/v0/latest",
+    "matlabReleaseInfoUrl": "https://mw-ci-static-dev.s3.amazonaws.com/matlab-release-info/",
```

Comment:
Is this temporary?

Reply:
Yes, wanted to make sure the integration tests passed. I wanted to try it with the 23b prerelease to make sure the versioning, toolcache, license, etc. worked as expected.